lib/OperatingSystems/FreeBSD.php

```diff
@@ -75,6 +75,17 @@ public function getCpuName(): string {
 		return $data;
 	}
 
+	public function getCpuCount(): int {
+		$numCpu = -1;
+
+		try {
+			$numCpu = intval($this->executeCommand('sysctl -n hw.ncpu')); //TODO: this should be tested if it actually works on FreeBSD
```

Comment:
That yields the expected result for my cpu.

Reply:
ok :+1: , then I will remove the comment